pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "pipreqs"
+version = "0.4.13"
+description = "Pip requirements.txt generator based on imports in project"
+authors = ["Vadim Kravcenko <vadim.kravcenko@gmail.com>"]
+license = "Apache-2.0"
+readme = ["README.rst", "HISTORY.rst"]
+packages = [{include = "pipreqs"}]
+repository = "https://github.com/bndr/pipreqs"
+keywords = ["pip", "requirements", "imports"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11"
+]
+
+[tool.poetry.scripts]
+pipreqs = "pipreqs.pipreqs:main"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<3.12"
```

Comment:
is the <3.12 because of that bug or because right now we only claim to support till 3.11?

Reply:
Supporting python 3.12 affects other files that are not relevant to this PR. It would be better to do it in a separate PR (spoiler: we are working on).